package.json

```diff
@@ -161,7 +161,6 @@
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@types/dateformat": "^3.0.1",
     "babel-plugin-styled-components": "^1.10.7",
-    "documentation": "12.3.0",
```

Comment:
What did you do about the examples and comments that documentation used to pull out? (If it was even working).

I still aim to have examples when we get time to make a better system for examples, comments, etc. but I think instead of parsing code it'll

Reply:
I moved them to the new `@traitsClass()` decorator
See https://github.com/TerriaJS/terriajs/pull/5713/files#discussion_r684918251

The JSDoc stuff wasn't working for me

Reply:
Yeah JSDoc doesn't like some ES2017 syntax, or something I can't remember what.